This patch makes some doctests pass 
but this will have to go when we install system-wide
and not in the user home.

--- a/sage/src/sage_setup/find.py
+++ b/sage/src/sage_setup/find.py
@@ -150,8 +150,8 @@
 
     EXAMPLES::
 
-        sage: from site import getsitepackages
-        sage: site_packages = getsitepackages()[0]
+        sage: from site import getusersitepackages
+        sage: site_packages = getusersitepackages()
         sage: files_by_module = installed_files_by_module(site_packages)
         sage: from sage.misc.sageinspect import loadable_module_extension
         sage: 'sage/structure/sage_object' + loadable_module_extension() in \
